### ISSUE TITLE
accumulated_num argument doesn't work

### ISSUE THREAD

**Issue by anselmwang** [yihong0618/bilingual_book_maker#359]
Hi, 

Thanks for creating this awesome tool. I'd like to use accumulated_num to reduce API cost.

However, the following command `python make_book.py --book_name test_books\animal_farm.epub --openai_key %openai_key% --accumulated_num 100 `  failed due to error "translate_list() takes 2 positional arguments but 3 were given"

Could you help to have a look?
Thanks a lot.

**Comment by yihong0618** [comment]
this only works for txt files

**Comment by anselmwang** [comment]
I am surprised. As I have fixed this bug by updating one line `result_txt_list = self.translate_model.translate_list(wait_p_list, context_flag)` to `result_txt_list = self.translate_model.translate_list(wait_p_list)`, just removing the last "context_flag"

And I have translated an EPub file, the final translated file looks perfect for me. 
Is it because that my input EPub file doesn't have much html tags?
